src/Illuminate/Foundation/Console/ClearCompiledCommand.php

```diff
@@ -25,17 +25,22 @@ class ClearCompiledCommand extends Command {
 	 */
 	public function fire()
 	{
-		$compiledPath = $this->laravel->getCachedCompilePath();
-		$servicesPath = $this->laravel->getCachedServicesPath();
-
-		if (file_exists($compiledPath))
+		if (method_exists($this->laravel,'getCachedCompilePath'))
 		{
-			@unlink($compiledPath);
+			$compiledPath = $this->laravel->getCachedCompilePath();	
+			if (file_exists($compiledPath))
+			{
+				@unlink($compiledPath);
+			}
 		}
-
-		if (file_exists($servicesPath))
+		
```

Comment:
Where do you see trailing whitespace? I'll close this and resubmit it with out it, so this change can go though.